universe/UniverseGenerator.h

```diff
@@ -81,7 +81,7 @@ void GenerateStarlanes(int max_jumps_between_systems, int max_starlane_length);
 bool SetEmpireHomeworld(Empire* empire, int planet_id, std::string species_name);
 
 /** Creates Empires objects for each entry in \a player_setup_data with
- * empire id equal to the specified player ids (so that the calling code
+ * empire id equal to the specified map key (so that the calling code
```

Comment:
empire ids equal to the corresponding map keys (...

Reply:
Fixed